src/destination.py

```diff
@@ -150,9 +151,9 @@ def __push_to_one_hermes_address(self, local_path, hermes_address):
             remote_connection.terminate()
 
         if return_code == 0:
-            print_err('Rsync successful.\n')
+            logging.info('Rsync successful.\n')
```

Comment:
It is, but the intention of '\n' was to create visual separation from further output, so that it's more "tied" with the preceding output of rsync command, but now with `logging` it seems not necessary.